src/main/webapp/app/exam/participate/summary/exercises/file-upload-exam-summary/file-upload-exam-summary.component.html

```diff
@@ -1,6 +1,9 @@
-<div *ngIf="submission">
+<div *ngIf="submission && submission.filePath; else noSubmission">
     <a id="downloadFileButton" (click)="downloadFile(submission.filePath!)">{{ 'artemisApp.fileUploadAssessment.submissionFile' | translate }}</a>
     <span class="ml-2 badge badge-info">
         {{ attachmentExtension(submission.filePath!) | uppercase }}
     </span>
 </div>
+<ng-template #noSubmission>
+    <div>No submission</div>
```

Comment:
Didn't test but this should probably be translated in german as well, right?

Reply:
No, the string that tells you that there is no submission comes directly from the parent component. This "no submission" string is not visible. I guess I could remove it.

Reply:
Ok as far as I have tested it, it's necessary for some weird reason to let the parent display the string correctly. Modeling and Text -exam-summary have the same thing, i adapted it from there

Reply:
Ok as long as it is not displayed i guess thats fine